common/scaffolding/reports2/text-level-en.html.html

```diff
@@ -0,0 +1,151 @@
+
+<html>
+<body>
+<nav>
+<b> (a element)</b>
+ <ul>
+  <li> <a href="/">aliquid</a> </li>
+  <li> <a href="/news">News</a> </li>
+  <li> <a>Examples</a> </li>
+  <li> <a href="/legal">Legal</a> </li>
+ </ul>
+</nav>
+
+<p> <b>(em element)</b>&nbsp;&nbsp;<em>Lorem ipsum dolor sit amet. Est expedita aliquid ea accusantium voluptatum et</em></P>
```

Comment:
"`<b>`" used here don't provide any sementic and are purely decorative. Used in this context are problematic.

Double non breaking space or any non breaking space is preferable to be avoided when not need or being use to separate an heading from its content like here. Also, non breaking space are read by AT software. Having two consecutive non breaking space `&nbsp;&nbsp;` provide the perception that the following failure technique are applicable: [F2 ](https://www.w3.org/WAI/WCAG22/Techniques/failures/F2) and [F43](https://www.w3.org/WAI/WCAG22/Techniques/failures/F43).

I added an example where the emphasis are added on a few words inside the paragraph which are very a common use.

```suggestion
<h2><code>em</code> element</h2>
<p><em>Lorem ipsum dolor sit amet. Est expedita aliquid ea accusantium voluptatum et</em></P>
<p>amet delectus <em>est</em> consequatur labore <em>vel atque deserunt</em> a sint animi ex voluptas</p>
```